README.md

```diff
@@ -9,8 +12,6 @@
 
 Jump-start into modern Python by forcing yourself to use f-strings.
 
-Works only with `python>=3.6`, apparently.
```

Comment:
Hmm I'm not sure why you want to remove this.. It still only work with python>=3.6 because 3.6 is when f-strings were introduced.

Reply:
The added PyPI badge shows which versions are supported. (Or strictly speaking, which versions are listed in the classifiers.)